src/Components/Components/src/PublicAPI.Shipped.txt

```diff
@@ -486,8 +486,6 @@ Microsoft.AspNetCore.Components.Routing.Router.NotFound.get -> Microsoft.AspNetC
 Microsoft.AspNetCore.Components.Routing.Router.NotFound.set -> void
 Microsoft.AspNetCore.Components.Routing.Router.OnNavigateAsync.get -> Microsoft.AspNetCore.Components.EventCallback<Microsoft.AspNetCore.Components.Routing.NavigationContext!>
 Microsoft.AspNetCore.Components.Routing.Router.OnNavigateAsync.set -> void
-Microsoft.AspNetCore.Components.Routing.Router.PreferExactMatches.get -> bool
-Microsoft.AspNetCore.Components.Routing.Router.PreferExactMatches.set -> void
```

Comment:
Do not modify *.Shipped.txt. Modify *.Unshipped.txt and add the *REMOVED* prefix to the API entry.